ios/engine/KMEI/KeymanEngine/Classes/Resource Management/ResourceDownloadManager.swift

```diff
@@ -423,6 +424,28 @@ public class ResourceDownloadManager {
     }
     return .upToDate
   }
+
+  /**
+   * A nice, centralized function designed to cleanly handle package-oriented downloads once the package's URL is known.
```

Comment:
Can you write this comment to describe the action `downloadPackage` takes? Is this going to be a public API or is it internal to Keyman Engine?

Reply:
This will be `public` - it's designed as the primary way for a framework consumer and/or UI code to download remote KMPs for use.  (Searching for them is a different matter.)